### PR TITLE
[naga] Naga Power Strategy

### PR DESCRIPTION
**Pull request by NuttakitDW**
Fixes # .

Changes proposed in this pull request:
- Adding naga folder in strategies folder
- Adding index.ts in naga folder
- Adding README.md folder in naga folder
- Adding naga strategy in index.js

### CODE REVIEW

**Comment by NuttakitDW** [comment]
Hi, did anything wrong with my PR?

File: src/strategies/index.ts [hunk 1]
```diff
@@ -108,6 +108,7 @@ import { strategy as aaveGovernancePower } from './aave-governance-power';
 import { strategy as cake } from './cake';
 import { strategy as planetFinance } from './planet-finance';
 import { strategy as impossibleFinance } from './impossible-finance';
+import { strategy as naga } from './naga';
```
Comment (ChaituVR) [inline src/strategies/index.ts:111]: Hi.. Please add `naga` to the exported object below, Thanks

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/naga/index.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      {
+        internalType: 'uint256',
+        name: 'rewardDebt',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
```
Comment (ChaituVR) [inline src/strategies/naga/index.ts:38]:
```suggestion
const masterChefAbi = [
  'function userInfo(uint256, address) view returns (uint256 amount, uint256 rewardDebt)'
]
```

File: src/strategies/index.ts (hunk 1 above)
Comment (NuttakitDW) [reply src/strategies/index.ts:111]: Thanks!!

File: src/strategies/naga/index.ts [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    ]),
+    { blockTag }
+  );
+
+  const sousBalances = await Promise.all(
+    options.chefAddresses.map((item) =>
+      multicall(
+        network,
+        provider,
+        sousChefabi,
```
Comment (ChaituVR) [inline src/strategies/naga/index.ts:77]: This is undefined.. please try running this strategy locally.. use command:

`npm run test --strategy=naga`

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/naga/index.ts (hunk 3 above)
Comment (NuttakitDW) [reply src/strategies/naga/index.ts:77]: Okie

**Pull request closed without merging**